README.md

```diff
@@ -10,15 +10,20 @@ This is a container image for running [Zulip](https://zulip.com)
 
 Current Zulip version: `6.1`
 Current Docker image version: `6.1-0`
+Current architectures supported: `amd64`
 
-Project status: **Alpha**. While this project works and is
-used by many sites in production, configuring is substantially more
-error-prone than the [normal Zulip installer][normal-install] (which
-Just Works). We recommend this project if you want to host Zulip
-using Docker, but both setting up and maintaining a Zulip server is
-simpler and less error-prone with the normal installer than with Docker.
+<!-- Remove when https://github.com/zulip/docker-zulip/issues/357 resolved -->
+> `arm64` support is experimental, and is not provided in the Docker Hub
+> images. To build an `arm64` image yourself, see `make help` locally.
```

Comment:
I would expect you don’t need any of this to build an arm64 image yourself. As long as you’re on an arm64 host, the usual `docker compose build` should build the image automatically. The only advantage of being on x86_64 is that you can pull our pre-built image from Docker Hub with `docker compose pull`.

Reply:
I guess the real selling point here is the cross-compilation: from an x86_64 machine (be that a dev laptop, or CI infrastructure), this allows building arm64 images for deploy to Graviton2, Raspberry Pi, whatever one's infra is. This is a stepping stone in my own eventual hopes for this repo: to fully automate our Docker Hub publishing when new Zulip Server releases are tagged. GitHub Actions is, last I saw, still fully on x86_64 and there aren't imminent plans to change that, so this x-compile tooling will be necessary for us to provide multiarch manifests for folks on such infrastructure to pull.

The linked #357 explicitly asks for us to start supporting ARM64 images on Docker Hub, which I think is a perfectly reasonable request given the spread of that hardware in the wild now.

Reply:
Absolutely. My comment is just about this specific documentation line.

Reply:
Aha - would something along these lines read more clearly to that end?

> `arm64` support is experimental, and is not provided in Docker Hub yet. On an arm64 host, the image can be build with the standard `docker build` flows; to cross-compile from an amd64 host, see `make help` locally.

Reply:
👍🏻 on that wording from me.

Reply:
This conversation superceded by the move to GHCR and fully-automated image builds.